Helper/Data.php

```diff
@@ -689,9 +695,17 @@ public function getClientKey($storeId = null)
     public function getWsUsername($storeId = null)
     {
         if ($this->isDemoMode($storeId)) {
-            $wsUsername = trim($this->getAdyenAbstractConfigData('ws_username_test', $storeId));
+            $wsUsernameTest = $this->getAdyenAbstractConfigData('ws_username_test', $storeId);
+            if (is_null($wsUsernameTest)) {
+                return false;
+            }
+            $wsUsername = trim($wsUsernameTest);
         } else {
-            $wsUsername = trim($this->getAdyenAbstractConfigData('ws_username_live', $storeId));
+            $wsUsernameLive = $this->getAdyenAbstractConfigData('ws_username_live', $storeId);
+            if (is_null($wsUsernameLive)) {
+                return false;
```

Comment:
Let's turn `string` configurations into `string|null` to represent lack of setup:
```suggestion
                return null;
```